recipes/gpi_core/yum_requirements.txt

```
@@ -0,0 +1,2 @@
++ xorg-x11-server-Xvfb
```

Glancing at the error testing it looks like you need to install OpenGL:
```suggestion
xorg-x11-server-Xvfb
mesa-libGL
mesa-dri-drivers
libselinux
libXdamage
libXxf86vm
```
See here: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=yum_requirements#libgl

---

We tried a few things in `yum_requirements` and weren't able to get things to build. I added these lines in the latest commit to show the resulting error.

---

Here's the error dump from the most recent failed test. The odd thing is that this seems to work fine if I include it as a command in `meta.yaml`.

<details>
```QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '\''/tmp/runtime-conda'\''
QApplication: invalid style override passed, ignoring it.
Fri Jul 26 20:59:05 2019 - gpi.loader:129 - ERROR - /home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/gpi_core/spinSim/GPI/SpinViz3D_GPI.py module failed to load in loadMod(.py) with:
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/gpi/loader.py", line 127, in loadMod
    mod = imp.load_module(store_name, fp, fullpath, description)
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/imp.py", line 234, in load_module
    return load_source(name, filename, file)
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 696, in _load
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/gpi_core/spinSim/GPI/SpinViz3D_GPI.py", line 41, in <module>
    import gpi.GLObjects as glo
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/gpi/GLObjects.py", line 60, in <module>
    import OpenGL, OpenGL.GL, OpenGL.GLU, OpenGL.GLUT
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/OpenGL/GL/__init__.py", line 4, in <module>
    from OpenGL.GL.VERSION.GL_1_1 import *
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/OpenGL/GL/VERSION/GL_1_1.py", line 14, in <module>
    from OpenGL.raw.GL.VERSION.GL_1_1 import *
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/OpenGL/raw/GL/VERSION/GL_1_1.py", line 7, in <module>
    from OpenGL.raw.GL import _errors
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/OpenGL/raw/GL/_errors.py", line 4, in <module>
    _error_checker = _ErrorChecker( _p, _p.GL.glGetError )
AttributeError: '\''NoneType'\'' object has no attribute '\''glGetError'\''

Fri Jul 26 20:59:05 2019 - gpi.library:208 - ERROR - Module not loadable, skipping. (/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/gpi_core/spinSim/GPI/SpinViz3D_GPI.py)
Fri Jul 26 20:59:05 2019 - gpi.loader:129 - ERROR - /home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/gpi_core/display/GPI/GLViewer_GPI.py module failed to load in loadMod(.py) with:
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/gpi/loader.py", line 127, in loadMod
    mod = imp.load_module(store_name, fp, fullpath, description)
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/imp.py", line 234, in load_module
    return load_source(name, filename, file)
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 696, in _load
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/gpi_core/display/GPI/GLViewer_GPI.py", line 46, in <module>
    import gpi.GLObjects as glo
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/gpi/GLObjects.py", line 60, in <module>
    import OpenGL, OpenGL.GL, OpenGL.GLU, OpenGL.GLUT
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/OpenGL/GL/__init__.py", line 4, in <module>
    from OpenGL.GL.VERSION.GL_1_1 import *
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/OpenGL/GL/VERSION/GL_1_1.py", line 14, in <module>
    from OpenGL.raw.GL.VERSION.GL_1_1 import *
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/OpenGL/raw/GL/VERSION/GL_1_1.py", line 7, in <module>
    from OpenGL.raw.GL import _errors
  File "/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/OpenGL/raw/GL/_errors.py", line 4, in <module>
    _error_checker = _ErrorChecker( _p, _p.GL.glGetError )
AttributeError: '\''NoneType'\'' object has no attribute '\''glGetError'\''

Fri Jul 26 20:59:05 2019 - gpi.library:208 - ERROR - Module not loadable, skipping. (/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.7/site-packages/gpi_core/display/GPI/GLViewer_GPI.py)
matplotlib version:  2.2.4
libgcc_s.so.1 must be installed for pthread_cancel to work
/home/conda/feedstock_root/build_artifacts/gpi_core_1564174456380/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/bin/gpi: line 50:  1468 Aborted                 (core dumped) $PYTHON $GPI_LAUNCH -style cleanlooks $@'
Tests failed for gpi_core-2.0.0rc-py37he1b5a44_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
WARNING conda_build.build:tests_failed(2230): Tests failed for gpi_core-2.0.0rc-py37he1b5a44_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
TESTS FAILED: gpi_core-2.0.0rc-py37he1b5a44_0.tar.bz2
```
</details>